extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -513,9 +562,11 @@ export class AmpAutocomplete extends AMP.BaseElement {
   renderResults_(filteredData, container, input) {
     let renderPromise = Promise.resolve();
     this.resetActiveElement_();
-    if (this.templateElement_) {
-      renderPromise = this.templates_
-        .renderTemplateArray(this.templateElement_, filteredData)
+    if (
+      this.templates_.hasTemplate(this.element, 'template, script[template]')
```

Comment:
This call is duplicated. Do we expect its return value to change?

Reply:
No, moved this to a variable